src/main.ts

```diff
@@ -16,8 +17,9 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import { ApiPromise } from '@polkadot/api';
-import { typesBundle, typesChain, typesSpec } from '@polkadot/apps-config/api';
+import * as apps from '@polkadot/apps-config/api';
```

Comment:
Whats the reason for importing the whole path here? I definitely think what you did looks a lot cleaner and way more readable.

Reply:
@TarikGul Just to clarify the difference between the keys with those names and the imports with those same names